content/articles/practicing-agility.md

```diff
@@ -0,0 +1,105 @@
+
+The Center has long thought deeply about how we do our work. We were guided
+first by the [Design Research methodology](http://ccnmtl.columbia.edu/dr/about/),
+which married “digital technologies with pedagogical theory and practice.” This
+iterative methodology ensured new projects aligned with our teaching and
+learning mission, but lacked detail on how the design and development
+work were completed. We borrowed elements from
+[Gov.UK’s Design Principles](https://www.gov.uk/guidance/government-design-principles)
+to fill in those gaps.
+But even then, we’ve still felt the need for a clearer implementation path.
+
+Our Associate Director [Shashi Yellambhatla](/authors/yellambhatla/) started us
+with the Scrum methodology early last year. We warmed up with regular stand-up
+meetings and two-week sprints. Daily standups started in the summer, along with
+a sprint review meeting. The Scrum introduction coincided with a tooling shift
+to Jira.
+
+This fall, we were ready to reflect and consider how to deepen our practice.
+We decided to engage in short readings and discussions on agile in general, and
+Scrum in particular. During these discussions, we all agreed that the
+principles expressed in the [Agile Manifesto](http://agilemanifesto.org/)
+reflect our core values. The stand-up meetings were mostly okay, and the sprint
+review meetings gave us a chance to demonstrate our work. But, we also pointed
+out rough spots...
+
+* Our team is small, and we are each working on at least two
+projects at a time. Scrum is organized around individual project teams.
+How do we tune this methodology so we can plan and prioritize work
+across projects?
+
+* Planning was still taking place at the individual level. Our multiple project
+backlogs were united...but each project team was entering tasks in different
+ways...or not at all. What was the “right” way to go from user stories to tasks?
+How could we integrate the project teams into this planning step?
+
+* Jira still felt awkward. We weren’t sure how to use epics, tasks, subtasks,
+components and other organization elements to reflect our work. We all expressed
+frustration with the interface, and the inability to see a solid view of our
+work in progress.
+
+What to do? Inspired by this [training video from Don McGreal](
+https://www.youtube.com/watch?v=syXnhduPxqM), we decided to take a break from
+Jira, and paper-prototype our planning and task creation process.
+
+Our project manager Meesha Meksin and I took the the job of collecting a set of
+user stories to prepare a sprint, and handwriting those stories on Post-it
+notes. In the first unified planning meeting, team members grabbed their user
+stories and wrote out their tasks. That first planning session went well,
+and took less time than the old meetings.
+
+A few quick reflections:
+
+* The process of backlog grooming and curating a set of user stories for a
+sprint actually doesn’t take that long. A few hours at the most. I was
```

Comment:
At what point is "a few hours" becomes "that long"? A couple, or three? Half a day?